engine/src/main/java/org/terasology/rendering/dag/tasks/SetWhatFacesToCullTask.java

```diff
@@ -17,15 +17,15 @@
 
 import org.lwjgl.opengl.GL11;
 import org.terasology.rendering.dag.RenderPipelineTask;
-import org.terasology.rendering.dag.stateChanges.SetCullFaceMode;
+import org.terasology.rendering.dag.stateChanges.SetFacesToCull;
 
 /**
  * TODO: Add javadocs
  */
-public final class SetCullFaceModeTask implements RenderPipelineTask {
+public final class SetWhatFacesToCullTask implements RenderPipelineTask {
```

Comment:
Really, this should be consistent with the associated state change. So, SetFacesToCull should be associated with SetFacesToCullTask. Sorry for suggesting this inconsistency, my fault.